Python/README.md

```diff
@@ -0,0 +1,25 @@
+# Rock, Paper, Scissors (Python)
+
+This is a simple Rock Paper Scissors game in Python
+
+## Installation
+
+Install Python in your editor. To install Python in Visual Studio Code, head to https://code.visualstudio.com/docs/python/python-tutorial
```

Comment:
I would instead recommend referencing the python.org documentation, since it's the source of truth. Like the other tutorials: https://github.com/zachflower/rock-paper-scissors/tree/master/c